src/CosmosDB/CosmosDB/help/New-AzCosmosDBSqlContainer.md

```diff
@@ -20,8 +20,8 @@ New-AzCosmosDBSqlContainer -ResourceGroupName <String> -AccountName <String> -Da
  [-AutoscaleMaxThroughput <Int32>] [-TtlInSeconds <Int32>] [-UniqueKeyPolicy <PSSqlUniqueKeyPolicy>]
  [-ConflictResolutionPolicyMode <String>] [-ConflictResolutionPolicyPath <String>]
  [-ConflictResolutionPolicyProcedure <String>] [-ConflictResolutionPolicy <PSSqlConflictResolutionPolicy>]
- [-AnalyticalStorageTtl <Int32>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+ [-ClientEncryptionPolicy <PSSqlClientEncryptionPolicy>] [-AnalyticalStorageTtl <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Could you please provide one example about how to create PSSqlClientEncryptionPolicy object for customer?

Reply:
the reason why i did not add is there is just an example on how to create a customer. But no example for the policies / optional parameters we support.

Reply:
It is nice to have an example to demonstrate the usage of complex parameters. We have had many cases that customer created a GitHub issue to ask how to use a complex parameter. But no worries, the example is not mandatory.

Reply:
Yup. Looks like i need to re-record all the tests. Will update the PR once all the tests run.

Reply:
fixed.